src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/MigrationQueueService.kt

```diff
@@ -52,7 +52,7 @@ class MigrationQueueService(
         .eventTypeMessageAttributes("prisoner-from-nomis-migration-${context.type.telemetryName}", noTracing = noTracing)
         .delaySeconds(delaySeconds)
         .build(),
-    ).thenAccept {
+    ).await().also {
```

Comment:
two separate issues here
1. We weren't waiting for the message to be sent, which is why I think the spans / threads were then getting mixed up
2. The `trackEvent` message was happening in a different future which meant that the `OperationId` etc. was missing